packages/fxa-shared/package.json

```diff
@@ -13,7 +13,9 @@
     "start": "pm2 start pm2.config.js",
     "stop": "pm2 stop pm2.config.js",
     "restart": "pm2 restart pm2.config.js",
-    "test": "tsc --build && node ./scripts/mocha-coverage.js -r ts-node/register --recursive test/**/*.{js,ts}",
+    "test": "yarn test:mocha && yarn test:jest",
+    "test:mocha": "tsc --build && node ./scripts/mocha-coverage.js -r ts-node/register --recursive test/**/*.{js,ts}",
+    "test:jest": "jest --coverage",
```

Comment:
I thought this would be more of a pain, but our structure of a `test` dir for the old tests, while Jest are in-line with the files using `*.spec.ts` actually made it pretty easy. This might be a good path for migrating other code to NestJS defaults like Jest in the future.